frontend/locales/en/common.ts

```diff
@@ -29,6 +29,28 @@ const common = {
     "repeat new password": "repeat new password",
     "new email": "new email",
     "repeat new email": "repeat new email",
+    "choose project name": "project name",
```

Comment:
```suggestion
    "project name": "project name",
```
.. This is true for others as well